.gitignore

```diff
@@ -1,2 +1,3 @@
 *.key
 *.retry
+secrets.yml
```

Comment:
Why no newline character at EOF?
https://robots.thoughtbot.com/no-newline-at-end-of-file

Reply:
good point ! will add them - actually I forgot to lint the files at all before I sent this, so let me do that.